txscript/sighash.go

```diff
@@ -330,8 +330,6 @@ func calcSignatureHash(prevOutScript []parsedOpcode, hashType SigHashType, tx *w
 			txOuts = nil
 		case SigHashSingle:
 			txOuts = tx.TxOut[:idx+1]
-		default:
```

Comment:
Please don't remove this.  It serves as documentation that unknown hash types act the same as `SigHashAll`.